### PR TITLE
doc: Project and Org API Key Support Clarification 

### PR DESCRIPTION
**Pull request by Zuhairahmed**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.
Project and Org level Key Support Clarification 

Link to any related issue(s): https://feedback.mongodb.com/forums/924145/suggestions/47453855 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code
- [ ] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

File: website/docs/r/access_list_api_key.html.markdown [hunk 1]
```diff
@@ -9,6 +9,8 @@ description: |-
 # Resource: mongodbatlas_project_ip_access_list_key
 
 `mongodbatlas_access_list_api_key` provides an IP Access List entry resource. The access list grants access from IPs, CIDRs or AWS Security Groups (if VPC Peering is enabled) to clusters within the Project.
+    
+-> **Note:** mongodbatlas_project_ip_access_list_key resource can be used to manage all Programmatic API Keys, regardless if they were created at the Organizational level or Project level.
```
Comment (davidhou17) [inline website/docs/r/access_list_api_key.html.markdown:13]:
```suggestion
-> **Note:** The `mongodbatlas_project_ip_access_list_key` resource can be used to manage all Programmatic API Keys, regardless if they were created at the Organizational level or Project level. 
```

**Review by davidhou17 — APPROVED** [review]
LGTM % 1 nit

File: website/docs/r/access_list_api_key.html.markdown (hunk 1 above)
Comment (davidhou17) [inline website/docs/r/access_list_api_key.html.markdown]: Code styling and grammar fix:
```suggestion
-> **Note:** The `mongodbatlas_project_ip_access_list_key` resource can be used to manage all Programmatic API Keys, regardless of whether they were created at the Organization level or Project level. 
```

File: website/docs/r/access_list_api_key.html.markdown [hunk 2]
```diff
@@ -9,6 +9,8 @@ description: |-
 # Resource: mongodbatlas_project_ip_access_list_key
 
 `mongodbatlas_access_list_api_key` provides an IP Access List entry resource. The access list grants access from IPs, CIDRs or AWS Security Groups (if VPC Peering is enabled) to clusters within the Project.
+    
+-> **Note:** The `mongodbatlas_project_ip_access_list_key` resource can be used to manage all Programmatic API Keys, regardless of whether they were created at the Organization level or Project level.
```
Comment (gssbzn) [inline website/docs/r/access_list_api_key.html.markdown:13]: [mongodbatlas_project_ip_access_list_key](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_ip_access_list) or [mongodbatlas_project_ip_access_list_key](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/access_list_api_key)?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**